### PR TITLE
Features/create ncne new task page

### PR DESCRIPTION
**Pull request by ssrajan**
NCNE new task page UI with mock data

### CODE REVIEW

File: src/NCNEPortal/Pages/Index.cshtml [hunk 1]
```diff
@@ -23,7 +21,7 @@
                 <button id="btnHistorical" class="btn btn-primary">Historical Tasks</button>
             </div>
             <div class="col-2">
-                <button id="btnNewTask" class="btn btn-primary">New Task</button>
+                <button id="btnNewTask" class="btn btn-primary" onclick="window.location.href = '/NewTask'">New Task</button>
```
Comment (UKHOGreg) [inline src/NCNEPortal/Pages/Index.cshtml]: `onclick="window.location.href = '/NewTask'"`
I would prefer that this inline onclick handler is instead implemented by a JQuery click handler.
e.g. 
`$("#btnNewTask").click(function(){
    window.location.href = '/NewTask';
})`

File: src/NCNEPortal/Pages/NewTask.cshtml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    <div class="col-4">
+        <h1>New Chart/New Edition</h1>
+    </div>
+    <div class="col-4">
+        <h1>New Task</h1>
+    </div>
+</div>
+<div class="col-12">
+    <div class="row justify-content-end btn-size">
+        <div class="col-2"><button id="btnClose" class="btn btn-primary btn-block" onclick="window.location.href = '/Index'">Close</button></div>
```
Comment (UKHOGreg) [inline src/NCNEPortal/Pages/NewTask.cshtml]: `onclick="window.location.href = '/Index'"`
I would prefer that this inline onclick handler is instead implemented by a JQuery click handler.
e.g.
`$("#btnClose").click(function(){
    window.location.href = '/Index';
})`

File: src/NCNEPortal/Pages/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -16,6 +17,7 @@
         <link rel="stylesheet" href="~/lib/font-awesome/css/all.min.css">
         <link rel="stylesheet" href="~/lib/bootstrap4/dist/css/bootstrap.min.css" />
         <link rel="stylesheet" href="~/lib/datatables/css/dataTables.bootstrap4.min.css" />
+        <link rel="stylesheet" href="~/lib/bootstrap-datepicker/css/bootstrap-datepicker.min.css" />
```
Comment (UKHOGreg) [inline src/NCNEPortal/Pages/Shared/_Layout.cshtml]: OK, this is possibly unnecessary, but can we bump this datepicker line above the datatables one?
Just so the order of execution matches the Development environment and there aren't any strange order of execution issues.

File: src/NCNEPortal/Pages/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -37,6 +40,7 @@
         <script src="~/lib/bootstrap4/dist/js/bootstrap.min.js"></script>
         <script src="~/lib/datatables/js/jquery.dataTables.min.js"></script>
         <script src="~/lib/datatables/js/dataTables.bootstrap4.min.js"></script>
+        <script src="~/lib/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
```
Comment (UKHOGreg) [inline src/NCNEPortal/Pages/Shared/_Layout.cshtml]: OK, this is possibly unnecessary, but can we bump this datepicker line above the datatables one?
Just so the order of execution matches the Development environment and there aren't any strange order of execution issues.

File: src/NCNEPortal/Pages/Index.cshtml (hunk 1 above)
Comment (Rossall82) [reply src/NCNEPortal/Pages/Index.cshtml]: Ha ha! I did this! We will also have to update the DbAssessment Portal then as this is where I copied and pasted from 🥇

Comment (SamirHassoun) [reply src/NCNEPortal/Pages/Index.cshtml]: or, the Button can be converted to <a> with href and make it look like a button, we did this in OWP 😊

Comment (ssrajan) [reply src/NCNEPortal/Pages/Index.cshtml]: changed to jquery

File: src/NCNEPortal/Pages/Shared/_Layout.cshtml (hunk 4 above)
Comment (ssrajan) [reply src/NCNEPortal/Pages/Shared/_Layout.cshtml]: Done

**Pull request merged**